k256/Cargo.toml

```diff
@@ -41,11 +41,12 @@ rand_core = { version = "0.6", features = ["getrandom"] }
 sha3 = { version = "0.10", default-features = false }
 
 [features]
-default = ["arithmetic", "ecdsa", "pkcs8", "schnorr", "std"]
+default = ["arithmetic", "basepoint-tables", "ecdsa", "once_cell", "pkcs8", "schnorr", "std"]
```

Comment:
"once_cell" would be already enabled by "basepoint-tables", right?

Reply:
Good call!